syntax = "proto3";

import "streamlit/proto/LabelVisibilityMessage.proto";

message TextArea {
  string id = 1;
  string label = 2;
  string default = 3;
  uint32 height = 4;
  uint32 max_chars = 5;
  string help = 6;
  string form_id = 7;
  string value = 8;
  bool set_value = 9;
  string placeholder = 10;
  bool disabled = 11;
  LabelVisibilityMessage label_visibility = 12;
}
